Snippets/Testing/Testing_6/MessageSession/MessageSessionTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Testing;
+using NUnit.Framework;
+
+[TestFixture]
+public class MessageSessionTests
+{
+    #region TestMessageSessionSend
+    [Test]
+    public async Task ShouldLogCorrectly()
+    {
+        var testableSession = new TestableMessageSession();
+
+        var somethingThatUseTheMessageSession = new SomethingThatUseTheMessageSession(testableSession);
+
+        await somethingThatUseTheMessageSession.DoSomething();
+
+        Assert.AreEqual(1, testableSession.SentMessages.Length);
+        Assert.IsInstanceOf<MyResponse>(testableSession.SentMessages[0].Message);
+    }
+    #endregion
+
+    class SomethingThatUseTheMessageSession
```

Comment:
should be `Uses`?

Reply:
Think both could work but no strong opinion?

Reply:
@timbussmann is correct, should be "uses"